core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.stories.ts

```diff
@@ -0,0 +1,28 @@
+import { moduleMetadata, Story, Meta } from '@storybook/angular';
+
+import { ButtonModule } from 'primeng/button';
+import { CardModule } from 'primeng/card';
+import { ScrollPanelModule } from 'primeng/scrollpanel';
+
+import { TemplateBuilderBoxComponent } from './template-builder-box.component';
+
+export default {
+    title: 'TemplateBuilderBoxComponent',
+    component: TemplateBuilderBoxComponent,
+    decorators: [
+        moduleMetadata({
+            imports: [ButtonModule, CardModule, ScrollPanelModule],
+            providers: []
+        })
+    ]
+} as Meta<TemplateBuilderBoxComponent>;
+
+const Template: Story<TemplateBuilderBoxComponent> = (args: TemplateBuilderBoxComponent) => ({
+    props: args
+});
+
+export const Primary = Template.bind({});
+
+Primary.args = {
+    size: 'large'
+};
```

Comment:
```suggestion
export const Small = Template.bind({});
export const Medium = Template.bind({});
export const Large = Template.bind({});

Small.args = {
    size: 'small'
};

Medium.args = {
    size: 'medium'
};

Large.args = {
    size: 'large'
};

```

This way we can have all three in this story.